muxers/yamux/src/lib.rs

```diff
@@ -146,23 +156,35 @@ where
 
 /// A stream produced by the yamux multiplexer.
 #[derive(Debug)]
-pub struct Stream(yamux::Stream);
+pub struct Stream(Either<yamux012::Stream, yamux013::Stream>);
+
+impl From<yamux012::Stream> for Stream {
```

Comment:
Don't do this, it leaks the `yamux` package into the public API :(

Reply:
:pray:   sneaky API leaks, you mentioned this before, thanks! :see_no_evil:   5bc711232